lib/cog_api/fake/bundles.ex

```diff
@@ -83,11 +84,33 @@ defmodule CogApi.Fake.Bundles do
 
   defp to_atom_keys(map) do
     Enum.reduce(map, %{}, fn ({key, val}, acc) ->
+      if is_map(val) do
```

Comment:
I don't think so. The `is_map(val)` part is independent of the `is_atom(key)` part below, so I'm not sure how a `cond` would clean it up.